firefox-ios/firefox-ios-tests/Tests/XCUITests/NewTabSettings.swift

```diff
@@ -129,6 +129,36 @@ class NewTabSettingsTest: BaseTestCase {
         validateKeyboardIsRaisedAndDismissed()
     }
 
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/2306875
+    // Smoketest
+    func testNewTabCustomURLKeyboardNotRaised() {
```

Comment:
Looks like it failes 3 out of 3 in Bitrise :(

Reply:
I am looking into it, there is a bug locally, investigation in progress.